css/main.css

```diff
@@ -1,3 +1,337 @@
 /* Add your custom CSS here */
 
+@import url('https://fonts.googleapis.com/css?family=Noto+Serif+SC&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+
 /* Dont' forget to link this file to your HTML in the <head> */
+
+body{
+    margin: 0;
+}
+.mainpage-section{
+    background-color: #c7c7c7;
+    background-image: url("/img/bg_laptop1200_1080.jpg"); 
+    min-height: 400px;
+    height: 400px;
+  
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: top center;
+    background-size: cover;
+   
+}
+
+.logo-wrapper{
+    display: flex;
+    justify-content: center;
+}
+
+.logo{
+    height: 80px;
+    width: auto;
+}
+
+nav{
+    font-family: 'Noto Serif SC', serif;
+    font-weight: 700;
+    font-size: 1.5em;
+}
+
+nav > ul {
+    display: flex;
+    flex-direction:row;
+    justify-content: center;
+    align-content: center;
+    flex-wrap: nowrap;
+    margin-top:15px;
+}
+
+nav > ul > li {
+    list-style-type: none;
+}
+
+nav a {
+    text-decoration: none;
+    cursor: pointer;
+}
```

Comment:
You could explore adding smooth scrolling for a better effect on click
https://css-tricks.com/almanac/properties/s/scroll-behavior/

Reply:
good suggestion